docs/spec/distribution/overview.md

```diff
@@ -2,71 +2,72 @@
 
 ## Overview
 
-This _simple_ distribution mechanism describes a functional way to passively 
-distribute rewards between validator and delegators. Note that this mechanism does 
-not distribute funds in as precisely as active reward distribution and will therefore
-be upgraded in the future. 
+This _simple_ distribution mechanism describes a functional way to passively
+distribute rewards between validators and delegators. Note, that this mechanism does
```

Comment:
Why do we want to add a comma after `Note`?

Reply:
Removed.